charts/internal/shoot-system-components/charts/csi-alicloud/values.yaml

```diff
@@ -6,7 +6,6 @@ images:
 credential:
   accessKeyID: keyID
   accessKeySecret: secret
-kubernetesVersion: v1.15.0
```

Comment:
Can we stop passing the `kubernetesVersion` in https://github.com/gardener/gardener-extension-provider-alicloud/blob/281cca7e059a4706d0ad685939702b85bce06d93/pkg/controller/controlplane/valuesprovider.go#L464-L479?